diameter_synthesis/simpler_diametrizer.py

```diff
@@ -60,9 +55,7 @@ def build_simpler_model(morphologies, config, fit_orders=None):
                             section_path_length(_section, cache) for _section in section.ipreorder()
                         )
                         lengths.append(
-                            tip_length
-                            - section_path_length(section, cache)
-                            + _s_length(section.id, section.points, cache)
+                            tip_length - section_path_length(section, cache) + section.length
```

Comment:
Did you benchmarked this new version vs your previous version? Because now the `setion.length` property is executed twice while it was only once in the previous version.

Reply:
yes, the slow part is the loops in section_path_lenght, I got from 6s to 0.2s by caching it instead of .length ;)

Reply:
Ah ok, cool :)